resotocore/resotocore/db/graphdb.py

```diff
@@ -1377,3 +1396,6 @@ async def copy_graph(self, to_graph: GraphName, to_snapshot: bool = False) -> Gr
             {"graph": self.graph_name, "to_graph": to_graph},
         )
         return await self.real.copy_graph(to_graph, to_snapshot=to_snapshot)
+
+    async def insert_usage_data(self, data: List[UsageDatapoint]) -> None:
```

Comment:
Should create two indexes:
- persistent index: fields=id,at, stored_values=v
- ttl index on at to let the values time out